repository-data/webfiles/src/main/resources/site/freemarker/common/searchresults.ftl

```diff
@@ -8,7 +8,7 @@
             <h1>No results for: ${query}</h1>
         <#else>
 
-            <h1>${pageable.total} result<#if pageable.total gt 1>s</#if> found</h2>
+            <h1 id="resultCount">${pageable.total} result<#if pageable.total gt 1>s</#if> found</h1>
```

Comment:
We started using attribute `data-uipath` in favour of ids for the purpose of identifying elements in acceptance tests. Would be good to apply it here; other `id` attributes in this template could use the same treatment, too.

Would be good to bring this one in line with that. Ask @wojtek-oledzki for details, it was his (good) idea :wink:

Reply:
Ah ok, will do